queries.sql

```diff
@@ -31,4 +31,57 @@ SELECT full_name as owner_full_name, name as animal_name FROM OWNERS
 SELECT full_name, COUNT(*) as total_animals FROM OWNERS 
   JOIN ANIMALS ON owners.id = animals.owner_id 
   GROUP BY FULL_NAME
-  ORDER BY total_animals DESC;
\ No newline at end of file
+  ORDER BY total_animals DESC;
+
+SELECT vets.name, visits.date_visited FROM VETS
+  JOIN VISITS ON visits.vet_id = vets.id
+  JOIN ANIMALS ON visits.animal_id = animals.id
+  WHERE vets.name = 'William Tatcher'
+  ORDER BY visits.date_visited DESC;
+
+SELECT vets.name as vet_name, species.specie_name, COUNT(animals.species_id) as species_count FROM VETS
+  JOIN VISITS ON visits.vet_id = vets.id
+  JOIN ANIMALS ON visits.animal_id = animals.id
+  JOIN SPECIES ON animals.species_id = species.id
+  WHERE vets.name = 'Stephanie Mendez'
+  GROUP BY animals.species_id, species.specie_name, vets.name;
+
+SELECT vets.name as vet_name, species.specie_name as specialized_specie FROM VETS
+  LEFT JOIN SPECIALTIES ON specialties.vet_id = vets.id
+  LEFT JOIN SPECIES ON specialties.species_id = species.id;
+
+SELECT animals.name as animal_name, vets.name as vet_name, visits.date_visited FROM VETS
+  JOIN VISITS ON visits.vet_id = vets.id
+  JOIN ANIMALS ON visits.animal_id = animals.id
+  WHERE vets.name = 'Stephanie Mendez' AND visits.date_visited >= '2020-04-01' AND visits.date_visited <= '2020-08-30'
+  ORDER BY visits.date_visited DESC;
+
+SELECT animals.name, COUNT(VISITS) as visits FROM ANIMALS
+  JOIN VISITS ON visits.animal_id = animals.id
+  GROUP BY animals.name
+  ORDER BY visits DESC;
+
+SELECT animals.name as animal_name, vets.name as vet_name, visits.date_visited FROM VETS
+  JOIN VISITS ON visits.vet_id = vets.id
+  JOIN ANIMALS ON visits.animal_id = animals.id
+  WHERE vets.name = 'Maisy Smith'
+  ORDER BY visits.date_visited DESC;
```

Comment:
- [ ] As pointed out above, you should also limit this in order to get the vet's first visit